app/src/main/java/edu/akelael/comics/model/ComicListModel.java

```diff
@@ -0,0 +1,48 @@
+package edu.akelael.comics.model;
+
+import java.util.HashMap;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+public class ComicListModel implements IComicListModel {
+    Server server;
+    Retrofit retrofit;
+    HashMap<String, String> mMap;
+
+    @Override
+    public void initializeApiConnection(){
+        Retrofit.Builder retrofitBuilder;
+        retrofitBuilder = new Retrofit.Builder();
+        retrofitBuilder.baseUrl(server.baseUrl);
+        retrofitBuilder.addConverterFactory(GsonConverterFactory.create());
+        retrofit = retrofitBuilder.build();
+        mMap = new HashMap<>();
+        mMap.put("ts", "ts");
+        mMap.put("apikey", "apikey");
+        mMap.put("hash", "hash");
+    }
+
+    @Override
+    public void getComics(final IComicListModelCallbacks listener){
+        server = retrofit.create(Server.class);
+        Call<Marvel> marvelCall = server.getCharacter(Server.ID_AMAZING_SPIDERMAN, mMap);
+        marvelCall.enqueue(new Callback<Marvel>() {
+            @Override
+            public void onResponse(Call<Marvel> call, Response<Marvel> response) {
+                if (response.code() == 200) {
```

Comment:
200 is what we call a "magic number". Someone reading this code might invest a lot of time trying to figure what the author tried to do (even if the author is you 3 months ago). In this case 200 is the OK code for an HTTP response, so we could make it a constant and name it RESPONSE_OK, for example. For this kind of refactor in Android Studio, you can position the cursor in the variable, and use Cmd + Alt + C in a Mac  (or Ctrl + Alt + C in a Windows).